modules/api/src/main/java/org/apache/ignite/Ignition.java

```diff
@@ -168,8 +170,15 @@ CompletableFuture<Ignite> start(
      * @param metaStorageNodeNames Names of nodes that will host the Meta Storage.
      * @param cmgNodeNames Names of nodes that will host the CMG.
      * @param clusterName Human-readable name of the cluster.
+     * @param authenticationConfig REST authentication configuration.
      * @throws IgniteException If the given node has not been started or has been stopped.
      * @see <a href="https://cwiki.apache.org/confluence/display/IGNITE/IEP-77%3A+Node+Join+Protocol+and+Initialization+for+Ignite+3">IEP-77</a>
      */
-    void init(String nodeName, Collection<String> metaStorageNodeNames, Collection<String> cmgNodeNames, String clusterName);
+    void init(
+            String nodeName,
```

Comment:
Looks like this list of parameters is becoming too large. I think we need to replace it with an object, that can be constructed using a builder. This way we will also be able to use some defaults, like disabled authentication and `cmgNodeNames` being equal to `metaStorageNodeNames`

Reply:
done